link_shortener/form_routes.py

```diff
@@ -26,14 +26,65 @@
 class CreateForm(SanicForm):
     endpoint = StringField('Endpoint', validators=[DataRequired()])
     url = StringField('URL', validators=[DataRequired()])
+    password = StringField('Password', validators=[DataRequired()])
     submit = SubmitField('Create')
 
 
 class UpdateForm(SanicForm):
     url = StringField('URL', validators=[DataRequired()])
+    password = StringField('Password', validators=[DataRequired()])
     submit = SubmitField('Update')
 
 
+class PasswordForm(SanicForm):
+    password = StringField('Password', validators=[DataRequired()])
+    submit = SubmitField('Submit')
+
+
+@form_blueprint.route('/authorize/<link_id>', methods=['GET'])
```

Comment:
Is it intentional that you have not add these routes into README?

Reply:
Updated